rasa/core/channels/console.py

```diff
@@ -111,7 +111,7 @@ def get_user_input(button_question: questionary.Question) -> Optional[Text]:
 async def record_messages(
     server_url=DEFAULT_SERVER_URL,
     auth_token="",
-    sender_id=UserMessage.DEFAULT_SENDER_ID,
+    sender_id=uuid.uuid4().hex,
```

Comment:
```suggestion
    sender_id=UserMessage.DEFAULT_SENDER_ID,
```
since it's set in the caller